index.html

```diff
@@ -2971,27 +2940,30 @@ <h3>Production</h3>
         <h3>Consumption</h3>
 
         <p>
-The <a>DID document</a> MUST be deserialized as a JSON document according to
-the <a href="#consumption">consumption rules for JSON</a>, with one additional
-requirement: The serialized <a>DID document</a> MUST include the
-<code>@context</code> member and be processed according to the rules below.
+All data structures expressed by a JSON-LD <a>representation</a> MUST be
+deserialized into the <a href="#data-model">data model</a> according to the JSON
+<a>representation</a> <a>consumption</a> rules as defined in <a
+href="#json"></a>.
         </p>
 
-        <dl>
-          <dt>@context</dt>
-          <dd>
-            <p>
-The value of the <code>@context</code> member conforms to the
-<a href="#production-0">JSON-LD Production Rules</a>. If more than one
-<a>URI</a> is provided, the <a>URIs</a> MUST be interpreted as an
-<a data-cite="INFRA#ordered-set">ordered set</a>.
-            </p>
-          </dd>
-        </dl>
+        <p>
+In addition to using the JSON <a>representation</a> <a>consumption</a> rules,
+JSON-LD consumption MUST add the representation-specific entries into a
+representation-specific <a href="#data-model">data model</a> according to the
```

Comment:
What's a "representation-specific data model"?

Reply:
I assume its a `data-model` with representation specific properties... for example an abstract data model with cbor integers for keys.

Reply:
@OR13's is one interpretation of how we could use the data model... except that it wouldn't work because we can't have integers as keys there.

I meant something slightly different, in an attempt to make your diagrams work @peacekeeper -- think of it as having two instantiations of the data model... one to hold the DID Document, and another to hold representation-specific properties. Your diagrams would then have two boxes that production to JSON-LD would pull from... the DID Document data model, and the representation-specific entries data model.

You could even make the case that if you see the same entry in the DID Document data model, and the representation-specific entries data model... and you don't recognize the one in representation-specific entries as something specific to your representation, then you don't need to serialize it. HOWEVER, I expect that @OR13 and I would object to that, UNLESS, the JSON representation noted that you have to preserve `@context` (because that's what we do in our implementations).

Reply:
if you don't want an `@context` don't consume JSON or JSON-LD that contains it.... craft your representation in the abstract and then produce ANY representation other than JSON-LD :)